### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by kabakTech**
[DEMO LINK](https://kabaktech.github.io/layout_stars/)
[TEST REPORT LINK](https://kabaktech.github.io/layout_stars/report/html_report/)

it fails the test but works fine. I managed to remove margin-right: 4px;

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
```
Comment (JoyCoffeeAddict) [inline src/style.css:9]: If you delete `width` and `justify-content`, then the effect will be almost the same, but this time correct. This is probably causing your problems with tests. Please read checklist again and apply point 3.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.star__wrapper {
```
Comment (JoyCoffeeAddict) [inline src/style.css:12]: You don't have such class in your html, so this selector is unnecessary.

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  display: flex;
+  width: 96px;
+}
+
+.stars__item {
```
Comment (JoyCoffeeAddict) [inline src/style.css:17]: You need to set fixed margin on each start to position them correctly.

**Review by JoyCoffeeAddict — CHANGES_REQUESTED** [review]

**Comment by kabakTech** [comment]
[DEMO LINK](https://kabaktech.github.io/layout_product-cards/)
[TEST REPORT LINK](https://kabaktech.github.io/layout_product-cards/report/html_report/)

**Pull request closed without merging**